.storybook/preview.tsx

```diff
@@ -42,7 +35,7 @@ const preview: Preview = {
       }
 
       return (
-        <div className={`${fontFigtree.variable} ${fontRecoleta.variable}`}>
+        <div className={`${fontBrittiSans.variable}`}>
```

Comment:
If we've only got one, I think we can do:
```suggestion
        <div className={fontBrittiSans.variable}>
```